tooltip.css

```diff
@@ -0,0 +1,116 @@
+body {
+    font-family: sans-serif; 
+    display: flex; /* Используем flexbox для центрирования содержимого */
+    justify-content: center; /* Выравнивание содержимого по горизонтали */
+    align-items: center; /* Выравнивание содержимого по вертикали */
+    height: 50vh; 
+}
+  
+/* Стили для контейнера с границей + анимация */
+#border {
+    border: 2px solid rgb(18, 57, 1); 
+    margin: auto; /* Центрируем по горизонтали */
+    padding: 20px; /* Отступы внутри контейнера */
+    text-align: center; 
+    opacity: 0; /* Изначально контейнер скрыт */
+    animation: slideFromLeft 0.8s linear forwards; /* linear - движение элемента происходит с постоянной скоростью */
+    animation-delay: 1s; /* Задержка в 1 секунду */
+}
+
+@keyframes slideFromLeft {
+    from {
+        transform: translateX(-100%);
+        opacity: 0;
+    }
+    to {
+        transform: translateX(0);
+        opacity: 1;
+    }
+}
+
+/* Стили для каждого триггера с тултипом */
+.tooltip {
+    position: relative; /* Позиционируем относительно своего родителя */
+    display: inline-block; /* Блочный элемент с возможностью выравнивания по горизонтали */
+    margin: 20px; /* Отступы вокруг элемента */
+    cursor: pointer; 
+    border: 2px solid rgb(167, 0, 0); 
+}
+
+.tooltip .tooltipleft, .tooltip .tooltiptop, .tooltip .tooltiptbottom, .tooltip .tooltipright {
```

Comment:
Good for optimizing the styles!

You could go even further with that and just add all of them a common selector, like ```.tooltip span {}``` and set these styles to each span that is in .tooltip

Or even better give a common class to these spans, so you will have for example ```<span class="tooltip_content tooltipleft">Tooltip Left</span>``` and set all common styles to class "tooltip_content"